src/components/AddTodoForm.jsx

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 
-const AddTodoForm = () => {
+const AddTodoForm = ({onAddTodo}) => {
+  /*
+    ============================
+    =         HANDLERS         =
+    ============================
+  */
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.title.value.trim();
+    /*
+      Validation:
+        - If todoTitle submitted is blank, then do not change the state.
+        Else, change the state.
+    */
+    if(!todoTitle){
+      return;
+    }
+    console.log(todoTitle);
+    onAddTodo(todoTitle);
+    event.target.reset();
+  };
```

Comment:
Excellent job here! You thought way ahead using trim() and checking if there's no todoTitle and exiting from the function! :1st_place_medal:

Reply:
Thank you! I really like validating forms haha!